src/asyoutypeformatter.js

```diff
@@ -206,31 +206,6 @@ i18n.phonenumbers.AsYouTypeFormatter.EMPTY_METADATA_
     .setInternationalPrefix('NA');
```

Comment:
The rest of the changes are updating the actual source code of the package

Reply:
These were automatic changes, right? Not manual?

Reply:
Automatic